core/admin/mailu/configuration.py

```diff
@@ -79,18 +81,26 @@ class ConfigManager(dict):
         'mysql': 'mysql://{DB_USER}:{DB_PW}@{DB_HOST}/{DB_NAME}'
     }
 
-    HOSTS = ('IMAP', 'POP3', 'AUTHSMTP', 'SMTP', 'REDIS')
-    OPTIONAL_HOSTS = ('WEBMAIL', 'ANTISPAM')
-
     def __init__(self):
         self.config = dict()
 
-    def resolve_host(self):
-        optional = [item for item in self.OPTIONAL_HOSTS if item in self.config and self.config[item] != "none"]
-        for item in list(self.HOSTS) + optional:
-            host = 'HOST_' + item
-            address = item + '_ADDRESS'
-            self.config[address] = system.resolve_address(self.config[host])
+    def get_host_address(self, name):
+        # if MYSERVICE_ADDRESS is defined, use this
+        if '{}_ADDRESS'.format(name) in os.environ:
+            return os.environ.get('%_ADDRESS' % name)
```

Comment:
Tested and looks like python doesn't like this syntax.
```
File "/app/mailu/configuration.py", line 119, in init_app
self.resolve_hosts()
File "/app/mailu/configuration.py", line 95, in resolve_hosts
self.config["IMAP_ADDRESS"] = self.get_host_address("IMAP")
File "/app/mailu/configuration.py", line 90, in get_host_address
return os.environ.get('%_ADDRESS' % name)
ValueError: unsupported format character '_' (0x5f) at index 1
```

Also tested only this function in a python shell and got same error

Reply:
@ionutfilip Thank you for pointing this out. Looks like I forgot this code when I reworked the PR